### DISCUSSION TITLE
implement remix style router

### DISCUSSION THREAD

**Discussion by Aksoom-Hussain** [nuxt/nuxt#16313] [Questions]
is there any way to implement remix style router on nuxt 3 page 

similar to 
https://www.npmjs.com/package/remix-router-vue

**Comment by wobsoriano** [comment]
Do you like the data loading and actions? 😄

**Reply by Aksoom-Hussain** [reply]
it very handy for spa apps but i am not react person to switch to remix, 
specially need for port my app to ionic capacitor app look to much like of web without that 
i know $fetch have the **loading** and **error** part but it not handy when i make fetch to plugin for all common routing with headers if there is any alternative to add the bearer token in all fetch request you are welcome for solution

**Reply by wobsoriano** [reply]
WIP https://github.com/wobsoriano/numix
